src/AndroidBridge.ts

```diff
@@ -0,0 +1,202 @@
+import { NativeEventEmitter, NativeModules } from 'react-native';
+import base64 from './utils/base64';
+import { IBridge } from './Bridge';
+import {
+  AppStore,
+  IProduct,
+  ITransactionGoogle,
+  ITransactionNativeGoogle,
+  TProductId
+} from './type';
+
+const { RNPaymentsGoogleModule: RNPayments } = NativeModules;
+
+export class AndroidBridge implements IBridge {
+  public eventEmitter = new NativeEventEmitter(RNPayments);
+
+  public static isAppStoreAvailable(): boolean {
+    return RNPayments && !!RNPayments.APP_STORE_AVAILABLE;
+  }
+
+  private static async open() {
+    await RNPayments.close();
+    await RNPayments.open();
+  }
+
+  availableAppStore(): AppStore {
+    return AndroidBridge.isAppStoreAvailable()
+      ? AppStore.GOOGLE
+      : AppStore.UNKNOWN;
+  }
+
+  async loadProducts(productIds: TProductId[]): Promise<IProduct[]> {
+    const validProductIds = productIds.filter(
+      p => p !== null && p !== undefined
```

Comment:
minor, but can we try to use words for arguments and variables rather than letters, just for readability purposes